src/components/Export.tsx

```diff
@@ -211,6 +227,70 @@ const ExportDialog: ExportDialogComponent = ({
       return { status: 500 };
     }
   };
+  const writeFileToIssue = async ({
+    title,
+    body,
+    setError,
+  }: {
+    title: string;
+    body: string;
+    setError: (error: string) => void;
+  }): Promise<{ status: number }> => {
+    try {
+      // https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#create-an-issue
+      const response = await apiPost({
+        domain: "https://api.github.com",
+        path: `repos/${selectedRepo}/issues`,
+        headers: {
+          Authorization: `token ${gitHubAccessToken}`,
+        },
+        data: {
+          title,
+          body,
```

Comment:
[oos] There will be formatting differences, like Roam bolding -> Github bolding.

This is where proxying through SamePage I think could be helpful, since we are defining the AtJson standard there to be interoperable across all text editors